### DISCUSSION TITLE
Insert/ Append/ Prepend a string to a File with a specific Name (at a specific Line)

### DISCUSSION THREAD

**Discussion by Excali6ur** [SilentVoid13/Templater#612] [Help] [answered]
Scouring the web I found a way to **prepend** to the _current File_ using `replaceRange`
`<%* window.app.workspace.activeLeaf.view.editor.replaceRange("---\n\n---\n", { line: 0, ch: 0 }, { line: 0, ch: 0 }) %>`
suggested by @liamcain 

But is there a way to either **insert** a string in a File with a _Specific Name_ at a specific line, or **append** to the end of that File?

**Comment by AB1908** [comment, marked as answer]
You can find the file to append to with `tp.file.find_tfile()` and then actually append to it with `await app.vault.append(TFile, "string contents")`.
